### PR TITLE
RFC Pattern - Adding Know Instance from DAZN

### PR DESCRIPTION
**Pull request by spier**
This [blog post ](https://medium.com/dazn-tech/building-a-dx-team-lessons-learned-4a66446088bc) by DAZN already contains more info about their RFC process than what we normally need for an org to be added as a Known Instance to a pattern.

This PR contains a summary of what the blog post shares about their RFC usage.

I am also informing DAZN about this PR, to see if they want to add anything else about their use of RFCs.

Part of #392

### CODE REVIEW

**Comment by spier** [comment]
Hi @loujaybee.

As your [blog post](https://medium.com/dazn-tech/building-a-dx-team-lessons-learned-4a66446088bc) already contains a fair amount of information about how DAZN uses RFCs, I would like to add DAZN as reference to the InnerSource pattern [Transparent Cross-Team Decision Making using RFCs](https://patterns.innersourcecommons.org/p/transparent-cross-team-decision-making-using-rfcs). 

FYI we call these references **Known Instances** i.e. an org that is implementing a given pattern.

Would you (or somebody else from DAZN) want to extend this description further, or are we good to merge this as is?

File: patterns/2-structured/transparent-cross-team-decision-making-using-rfcs.md [hunk 1]
```diff
@@ -97,6 +97,7 @@ Also for decision making outside of pure software design decisions, transparent
 - **Europace** - As described in Open Organization: [Setting cross-team standards and best practices in the open][europace].
 - **Uber** - According to this blog post by Gergely Orosz: [Scaling Engineering Teams via RFCs: Writing Things Down][uber].
 - **Google Design Docs** - As described in this blog post by Malte Ubl [Design Docs at Google][google]
+- **DAZN** - One way that DAZN makes technical decisions is via RFCs. RFCs are used for decisions that apply to engineering-wide processes only! The RFCs live in a GitHub repository, and technical standards are then gradually adopted within our tools and by our engineers in the company. An RFC can be raised by any engineer, and voted on by all engineers. If upvotes exceed downvotes, the RFC is adopted. It’s worth noting, that the RFC voting process hasn’t yet been “stress-tested” by any contentious decisions. - As described in this blog post by Lou Bichard: [Building A DX Team: Lessons Learned][dazn]
```
Comment (spier) [inline patterns/2-structured/transparent-cross-team-decision-making-using-rfcs.md]:
```suggestion
- **DAZN** - One way that DAZN makes technical decisions is via RFCs. RFCs are used for decisions that apply to engineering-wide processes only! The RFCs live in a GitHub repository, and technical standards are then gradually adopted within their tools and by their engineers. An RFC can be raised by any engineer, and voted on by all engineers. If upvotes exceed downvotes, the RFC is adopted. It’s worth noting, that the RFC voting process hasn’t yet been “stress-tested” by any contentious decisions. - As described in this blog post by Lou Bichard: [Building A DX Team: Lessons Learned][dazn]
```

Changing the grammar as we are talking about DAZN here, and not they about themselves.

**Comment by loujaybee** [comment]
@nodefortytwo is really your person, however the content looks good to me.

**Comment by spier** [comment]
hi @nodefortytwo would you be happy for us to publish this info about DAZN uses RFCs internally in our InnerSource pattern here? https://patterns.innersourcecommons.org/p/transparent-cross-team-decision-making-using-rfcs

File: patterns/2-structured/transparent-cross-team-decision-making-using-rfcs.md [hunk 2]
```diff
@@ -97,6 +97,7 @@ Also for decision making outside of pure software design decisions, transparent
 - **Europace** - As described in Open Organization: [Setting cross-team standards and best practices in the open][europace].
 - **Uber** - According to this blog post by Gergely Orosz: [Scaling Engineering Teams via RFCs: Writing Things Down][uber].
 - **Google Design Docs** - As described in this blog post by Malte Ubl [Design Docs at Google][google]
+- **DAZN** - One way that DAZN makes technical decisions is via RFCs. RFCs are used for decisions that apply to engineering-wide processes only! The RFCs live in a GitHub repository, and technical standards are then gradually adopted within their tools and by their engineers. An RFC can be raised by any engineer, and voted on by all engineers. If upvotes exceed downvotes, the RFC is adopted. It’s worth noting, that the RFC voting process hasn’t yet been “stress-tested” by any contentious decisions. - As described in this blog post by Lou Bichard: [Building A DX Team: Lessons Learned][dazn]
```
Comment (spier) [inline patterns/2-structured/transparent-cross-team-decision-making-using-rfcs.md]:
```suggestion
- **DAZN** (10/2021) - One way that DAZN makes technical decisions is via RFCs. RFCs are used for decisions that apply to engineering-wide processes only! The RFCs live in a GitHub repository, and technical standards are then gradually adopted within their tools and by their engineers. An RFC can be raised by any engineer, and voted on by all engineers. If upvotes exceed downvotes, the RFC is adopted. It’s worth noting, that the RFC voting process hasn’t yet been “stress-tested” by any contentious decisions. - As described in this blog post by Lou Bichard: [Building A DX Team: Lessons Learned][dazn]
```

Adding the date of the blog post to the Known Instance.

**Comment by spier** [comment]
As this was already shared by DAZN publicly in 10/2021, I am assume we are fine to include this in our pattern.
I am going ahead and merging this.

**Pull request merged**